src/ImageProcessorCore/IO/EndianBinaryReader.cs

```diff
@@ -122,7 +122,7 @@ public void Close()
         /// </summary>
         /// <param name="offset">Offset to seek to.</param>
         /// <param name="origin">Origin of seek operation.</param>
-        public void Seek(int offset, SeekOrigin origin)
+        public virtual void Seek(int offset, SeekOrigin origin)
```

Comment:
Since the TIFF reader can be used inside a JPEG and the beginning of the TIFF image is not the beginning of the stream. I made this virtual and have overridden it in the [TiffReader](https://github.com/stbraley/ImageProcessor/blob/Core/src/ImageProcessorCore/Formats/Tiff/TiffReader.cs) class to move to the beginning of the TIFF file. I'm not sure this was your intent for this class.